include/cutlass/uint128.h

```diff
@@ -56,7 +56,7 @@
 #include "cutlass/cutlass.h"
 
 /// Optionally enable GCC's built-in type
-#if (defined(__x86_64) || defined (__aarch64__)) && !(defined(__CUDA_ARCH__) && ((__CUDACC_VER_MAJOR__ <= 10) || ((__CUDACC_VER_MAJOR__ == 11) && (__CUDACC_VER_MINOR__ <= 4)))) && defined(__GNUC__) && !defined(CUTLASS_ENABLE_SYCL)
+#if (defined(__x86_64) || defined (__aarch64__)) && !((defined(__CUDA_ARCH__) || defined(__SYCL_CUDA_ARCH__)) && ((__CUDACC_VER_MAJOR__ <= 10) || ((__CUDACC_VER_MAJOR__ == 11) && (__CUDACC_VER_MINOR__ <= 4)))) && defined(__GNUC__)
```

Comment:
is this only for __SYCL_CUDA_ARCH__? I dont think so. I think this is for GPUs including the Intel. So it should be __SYCL_DEVICE_ONLY__

Reply:
That's a good point, committed your suggestion verbatim